### ISSUE TITLE
Multi class segmentation

### ISSUE THREAD

**Issue by aihyper11** [zlkanata/DeepGlobe-Road-Extraction-Challenge#29]
What changes are requited in the current script to use for multi class problem?

**Comment by zlckanata** [comment]
This repo is rough and only suitable for binary seg now, I guess it is necessary to edit several files for multi-class problem.

Dataloader(label generation)
https://github.com/zlkanata/DeepGlobe-Road-Extraction-Challenge/blob/master/data.py#L91

Loss Function
https://github.com/zlkanata/DeepGlobe-Road-Extraction-Challenge/blob/mastre/loss.py#L7

Network's Last Layer
https://github.com/zlkanata/DeepGlobe-Road-Extraction-Challenge/blob/master/framework.py#L10
self.net = net().cuda() --> self.net = net(num_class=?).cuda()

Evaluation
https://github.com/zlkanata/DeepGlobe-Road-Extraction-Challenge/blob/master/test.py

**Comment by aihyper11** [comment]
Thank you very much for your reply. I will go through all the suggested changes.
